language_names/manual.sql

```diff
@@ -1,482 +0,0 @@
-INSERT INTO "fixes" VALUES(NULL,'ca','srd','sard');
-INSERT INTO "fixes" VALUES(NULL,'en','srd','Sardinian');
-INSERT INTO "fixes" VALUES(NULL,'it','srd','sardo');
-INSERT INTO "fixes" VALUES(NULL,'es','srd','sardo');
-INSERT INTO "fixes" VALUES(NULL,'eu','crh','Krimeako tatarera');
-INSERT INTO "fixes" VALUES(NULL,'en','oci_aran','Occitan Aranese');
-INSERT INTO "fixes" VALUES(NULL,'de','oci_aran','Okzitanisch Aranesisch');
-INSERT INTO "fixes" VALUES(NULL,'oci_aran','oci_aran','Aranés');
-INSERT INTO "fixes" VALUES(NULL,'kmr','kmr','Kurmancî');
-INSERT INTO "fixes" VALUES(NULL,'en','kmr','Kurdish Kurmanji');
-INSERT INTO "fixes" VALUES(NULL,'scn','scn','sicilianu');
-INSERT INTO "fixes" VALUES(NULL,'es','scn','siciliano');
-INSERT INTO "fixes" VALUES(NULL,'ca','scn','sicilià');
-INSERT INTO "fixes" VALUES(NULL,'en','scn','Sicilian');
-
-INSERT INTO "fixes" VALUES(NULL,'en','smn','Inari Saami');
-INSERT INTO "fixes" VALUES(NULL,'fi','smn','inarinsaame');
-INSERT INTO "fixes" VALUES(NULL,'nn','smn','enaresamisk');
-INSERT INTO "fixes" VALUES(NULL,'nb','smn','enaresamisk');
-INSERT INTO "fixes" VALUES(NULL,'sma','smn','enaresaemien');
-INSERT INTO "fixes" VALUES(NULL,'se','smn','anárašgiella');
-INSERT INTO "fixes" VALUES(NULL,'smj','smn','anárašgiella');
-INSERT INTO "fixes" VALUES(NULL,'smn','smn','anarâškielâ');
-
-INSERT INTO "fixes" VALUES(NULL,'en','smj','Lule Saami');
-INSERT INTO "fixes" VALUES(NULL,'fi','smj','luulajansaame');
-INSERT INTO "fixes" VALUES(NULL,'nn','smj','lulesamisk');
-INSERT INTO "fixes" VALUES(NULL,'nb','smj','lulesamisk');
-INSERT INTO "fixes" VALUES(NULL,'sma','smj','julevsaemiengïele');
-INSERT INTO "fixes" VALUES(NULL,'se','smj','julevsámegiella');
-INSERT INTO "fixes" VALUES(NULL,'smj','smj','julevsámegiella');
-INSERT INTO "fixes" VALUES(NULL,'smn','smj','juulevsämikielâ');
-
-INSERT INTO "fixes" VALUES(NULL,'sma','se','noerhtesaemiengïele');
-INSERT INTO "fixes" VALUES(NULL,'smj','se','nuorttasámegiella');
-INSERT INTO "fixes" VALUES(NULL,'smn','se','pajekielâ');
-
-INSERT INTO "fixes" VALUES(NULL,'sma','sma','åarjelsaemiengïele');
-INSERT INTO "fixes" VALUES(NULL,'smj','sma','oarjjelsámegiella');
-INSERT INTO "fixes" VALUES(NULL,'smn','sma','maadâsämikielâ');
-
-INSERT INTO "fixes" VALUES(NULL,'sma','no','daaroegïele');
-INSERT INTO "fixes" VALUES(NULL,'smj','no','dárogiella');
-INSERT INTO "fixes" VALUES(NULL,'smn','no','tárukielâ');
-
-INSERT INTO "fixes" VALUES(NULL,'sma','fi','såevmiengïele');
-INSERT INTO "fixes" VALUES(NULL,'smj','fi','suomagiella');
-INSERT INTO "fixes" VALUES(NULL,'smn','fi','suomâkielâ');
-
-INSERT INTO "fixes" VALUES(NULL,'tr','gag','Gagavuzca');
-INSERT INTO "fixes" VALUES(NULL,'gag','gag','Gagauzça');
-INSERT INTO "fixes" VALUES(NULL,'crh','gag','ğağauzça');
-INSERT INTO "fixes" VALUES(NULL,'kk','gag','ғағауызша');
-INSERT INTO "fixes" VALUES(NULL,'ky','gag','гагаузча');
-
-INSERT INTO "additions" VALUES(NULL,'en','lvs','Latvian');
```

Comment:
Do you think we could move these additions to a separate TSV?

Reply:
Originally I had separate files fixes.tsv and additions.tsv. I merged them as you wanted to keep the number of files same. I can split them again.

Reply:
Ah, sorry! That's my bad. I didn't want multiple versions of the same scraped file or multiple turkic files. The additions and fixes are good since they represent semantically different things.

Reply:
Okay I revert these changes